MapboxSearch/sdk/src/main/java/com/mapbox/search/analytics/FeedbackEvent.kt

```diff
@@ -137,6 +137,16 @@ public class FeedbackEvent @JvmOverloads public constructor(
              */
             public const val INCORRECT_LOCATION: String = "incorrect_location"
 
+            /**
+             * Feedback reason for incorrect POI phone number.
+             */
+            public const val INCORRECT_PHONE_NUMBER: String = "incorrect_phone_number"
```

Comment:
[nit]

Please note, that new values are not added to the `@StringDef` declaration:

```kotlin
@Retention(AnnotationRetention.SOURCE)
@StringDef(
      INCORRECT_NAME,
      INCORRECT_ADDRESS,
      INCORRECT_LOCATION,
      INCORRECT_PHONE_NUMBER,
      INCORRECT_RESULT_RANK,
      OTHER,
)
```

Reply:
Thanks, fixed :)